controllers/etcd_controller_test.go

```diff
@@ -2462,6 +2623,37 @@ func getEtcd(name, namespace string, tlsEnabled bool) *druidv1alpha1.Etcd {
 	return instance
 }
 
+func getEtcdWithReplicas(name, namespace string, replicas int) *druidv1alpha1.Etcd {
```

Comment:
Would it be easier to just call `getEtcdWithDefault` and update the replicas here?

Reply:
Done